libs/sdk/src/RudderClient.ts

```diff
@@ -217,34 +217,33 @@ async function group(
   bridge.group(groupId, filterNaN(traits), filterNaN(options));
 }
 
-// wrapper for `alias` method
-async function alias(newId: string, options?: Record<string, unknown> | null): Promise<void>;
-/**
- * @deprecated use alias{@link alias(newId: string, options?: Record<string, unknown> | null)} instead
- */
-async function alias(previousId: string, userId: string | Record<string, unknown>): Promise<void>;
 async function alias(
```

Comment:
We still need to add these overloaded signatures above this function definition to support function overloading.

```
// wrapper for `alias` method
async function alias(newId: string, options?: Record<string, unknown> | null): Promise<void>;
async function alias(newId: string, previousId: string, options?: Record<string, unknown> | null): Promise<void>;
```